_data/trans/vi.json

```diff
@@ -1,20 +1,20 @@
 {
-    "about": "About",
-    "contribute": "Fork on GitHub",
+    "about": "Giới thiệu",
+    "contribute": "Fork trên GitHub",
```

Comment:
This became `"contribute": "Contribute on GitHub"` on the source language, does this take it into consideration?